R/nix_build.R

```diff
@@ -22,7 +20,7 @@
 #' \dontrun{
 #' nix_build()
 #' }
-nix_build <- function(project_path = ".",
+nix_build <- function(project_path,
```

Comment:
```suggestion
nix_build <- function(project_path = getwd(),
```

Reply:
I think we can keep it, it makes it clear that we expect a directory at least, and not the full path to a `default.nix` file